FeedStoreChallenge/UserDefaultsFeedStore.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  UserDefaultsFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Omar Bassyouni on 24/01/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public final class UserDefaultsFeedStore {
+	
+	private let userDefaults: UserDefaults
+	private let queue = DispatchQueue(label: "\(UserDefaultsFeedStore.self)Queue",
+									  qos: .userInitiated,
+									  attributes: .concurrent)
+	
+	private var feedCacheKey: String {
+		"feed_cache_key"
+	}
+	
+	public init(_ userDefaults: UserDefaults = UserDefaults.standard) {
```

Comment:
This line can be shorter:

```suggestion
	public init(_ userDefaults: UserDefaults = .standard) {
```